WireLogging/Sources/WireLogging/LogAttributes.swift

```diff
@@ -25,6 +25,8 @@ public enum LogAttributesKey: String, Comparable, Sendable {
     case recipientID = "recipient_id"
     case eventId = "event_id"
     case eventEnvelopeID = "event_envelope_id"
+    case ackMultipleEventsCount = "ack_events_count"
+    case multipleEvents = "ack_multiple_events"    
```

Comment:
this one represents the bool value we pass to acknowledgement either true or false